tools/build-env.sh

```diff
@@ -14,4 +14,4 @@ export PKGDEST=${PKGDEST:-/tmp/build}
 export LOGDEST=${LOGDEST:-$PKGDEST}
 export SOURCE_DATE_EPOCH=${DRONE_BUILD_CREATED:-${DRONE_BUILD_STARTED:-$(date +%s)}}
 mkdir -p .tmp/pkgs
-alias run_nobody='PACKAGES="$(cd .tmp/pkgs; ls -1)" sudo -u nobody --preserve-env=CI,PACKAGER,PKGDEST,LOGDEST,SRCDEST,SRCPKGDEST,BUILDDIR,PACKAGE,PACKAGES,SOURCE_DATE_EPOCH,IGNOREDB,DRONE_TARGET_BRANCH,MAKEPKG_ARGS'
```

Comment:
We could also consider setting `MAKEFLAGS` dynamically in this file to e.g. `$[$(nproc) + 1]`. But we can eave this to a later PR.